stories/Select.tsx

```diff
@@ -727,10 +728,14 @@ storiesOf('Organisms|Select Components', module)
                         inputRef={inputRef}
                         // this will trick to UI to pre-fill the in input field when it gets focused again
                         initInputValue={store.get('singleSelectedText')}
-                        // Here we switch the UI behavior based on our local state
+                        // here we switch the UI behavior based on our local state
                         customSelectionIndicator={
                             store.get('singleSelectedText') ? customBlur : undefined
                         }
+                        // select the first suggestions (user input) on outer click
+                        selectFirstOnOutClick={
+                            !!inputValue.length && !store.get('selectedSuggestions').length
```

Comment:
Should this logic live inside the Dropdown component?

Reply:
no, it is business logic not UI logic

Reply:
Not clear to me why it is business logic. Can It be the case when we need to select the first item when the input value doesn't exist?

Reply:
I guess you have a point. We already hard code selecting the first item, so we can include this as well.